packages/ra-ui-materialui/src/list/BulkActionsToolbar.js

```diff
@@ -5,12 +5,15 @@ import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
 import { lighten } from '@material-ui/core/styles/colorManipulator';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
 import { useTranslate, sanitizeListRestProps } from 'ra-core';
 
 import TopToolbar from '../layout/TopToolbar';
 
 const useStyles = makeStyles(theme => ({
     toolbar: {
+        display: 'flex',
```

Comment:
Why do you need to do that? Can't you just add the "Unselect" link in the div of class title?

Reply:
> Can't you just add the "Unselect" link in the div of class title?

I tried, but vertical alignment was off.

So I added `IconButton` as a sibling before the title div, but the effect was that the IconButton got 1/3 of the width of the entire toolbar, which I fixed with toolbar.display='flex' and title.flex=1 instead of title.flex='0 0 auto'.